src/__tests__/suggestions.js

```diff
@@ -103,6 +103,15 @@ test('should not suggest when suggest is turned off for a query', () => {
   ).not.toThrowError()
 })
 
+test('should suggest when suggest is turned on for a specific query but disabled in config', () => {
+  configure({throwSuggestions: false})
+  renderIntoDocument(`
+  <button data-testid="foo">submit</button>
+  <button data-testid="foot">another</button>`)
+
+  expect(() => screen.getByTestId('foo', {suggest: true})).toThrowError()
```

Comment:
This isn't really testing whether we're suggesting anything. Just that an error is thrown. I'll see if a snapshot makes more sense